src/i18n/locales/en/translation.json

```diff
@@ -890,6 +890,13 @@
       "time_estimate": "ℹ️ Transactions usually arrive within 10-30 minutes.",
       "service_unavailable": "Service temporary not available, please try again in a few minutes."
     },
+    "confirm_payment_async": {
+      "title": "Approve Delayed / Conditional Payment",
+      "description": "This payment may stay unsettled for a day or longer. The receiver can choose to settle or cancel the payment by leaving it to expire.",
```

Comment:
description should be: "This is a HODL invoice. Learn more >>" with a link to an Alby guides page for HODL invoices.

Reply:
is it "HODL" or "HOLD"? I think in the docs it is mostly called "hold", isn't it?